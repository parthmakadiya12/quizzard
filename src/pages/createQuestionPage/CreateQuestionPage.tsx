import { H3, H4, P } from "../../common/styles/typography";
import styled from "styled-components";

const CreateQuestionPage = () => {
  return (
    <Wrapper>
      <H3>Create New Question</H3>
      <form>
        <H4>Question</H4>
      </form>
    </Wrapper>
  );
};

const Wrapper = styled.div`
  margin: auto 1rem;
`;

export default CreateQuestionPage;
