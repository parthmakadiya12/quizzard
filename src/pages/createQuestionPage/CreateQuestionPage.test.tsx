import CreateQuestionPage from "./CreateQuestionPage";
import renderer from "react-test-renderer";

describe("<CreateQuestionPage>", () => {
  let wrapper: any;

  it("should render page correctly", () => {
    wrapper = renderer.create(<CreateQuestionPage />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
