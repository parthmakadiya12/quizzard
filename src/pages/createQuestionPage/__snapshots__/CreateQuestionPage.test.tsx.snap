// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CreateQuestionPage> should render page correctly 1`] = `
<div
  className="sc-jSFjdj fLRfQM"
>
  <h3
    className="sc-dlnjwi gLtfcp"
  >
    Create New Question
  </h3>
  <form>
    <h4
      className="sc-hKFxyN ezUaUl"
    >
      Question
    </h4>
  </form>
</div>
`;
